client/src/modules/Interviews/pages/InterviewFeedback/index.tsx

```diff
@@ -18,12 +17,11 @@ type FormAnswer = {
   answer: string;
 };
 
-export function InterviewFeedback({ course, type, interviewId }: PageProps) {
+export function InterviewFeedback({ course, type, interviewTaskId, githubId }: PageProps) {
   const courseId = course.id;
-  const router = useRouter();
   const session = useContext(SessionContext);
-  const template = templates[type];
-  const githubId = router.query.githubId as string;
+
+  const template = templates[type as keyof typeof templates];
```

Comment:
I think you don't need a type assertion here

Reply:
at this version you are right, but I have more changes coming and this casting will be needed. but thanks for your attention to details.

Reply:
nvm, I fixed it